rebar.config.sample

```diff
@@ -42,6 +42,16 @@
 %% yecc files to compile first
 {yrl_first_files, []}.
 
+%% == Application Discovery ==
+
+%% Extensions for discovering application resource files. Rebar3 discovers
+%% applications by searching for these files. When building the application
+%% information, rebar will use the first discovered files in the list.
+%% This configuration is intended to let profiles compose with special app
+%% files by prefixing extensions.
+{application_resource_extensions, [
+    ".app.src.script", ".app.src", ".exs"
```

Comment:
`.exs` is not an extension specific to an application, it is more like `.escript`. So I don't think should exist in this list.

Reply:
it's part of the resolving we do and see as valid, weirdly enough I think

Reply:
Thanks for clarifying this, I was wondering how mix.exs should be handled. I'll special case it similar to the plain .app file.